sdks/go/examples/wordcount/wordcount.go

```diff
@@ -104,9 +104,11 @@ var (
 var wordRE = regexp.MustCompile(`[a-zA-Z]+('[a-z])?`)
 
 // extractFn is a DoFn that emits the words in a given line.
-func extractFn(line string, emit func(string)) {
+func extractFn(ctx context.Context, line string, emit func(string)) {
+	c := beam.GetCounter("extract", "count")
```

Comment:
Done. I'm going to defer updating the other examples to match java to another PR to avoid too much churn.

As a rule we do want to declare metrics globally. It was to demonstrate/sanity check that local definitions with the same namespace.name are "the same" counter. This is now handled in the declareAnywhere example.